README.md

```diff
@@ -141,6 +141,87 @@ SearchResponse response = constructor.naturalLanguageSearch(request, userInfo);
 String response = constructor.naturalLanguageSearchAsJSON(request, userInfo);
 ```
 
+# Retrieving Browse Results
+
+To retrieve browse results, you will need to create a `BrowseRequest`. When creating the `BrowseRequest` the filter name can be one of `collection_id`, `group_id`, or a facet name (i.e. Brand). In this request, you can also specify the number of results you want per page, the page you want, sorting instructions, and also filter the search by category or facets. If the results are for a specific user, you can also create a `UserInfo` object, which will allow you to retrieve personalized results.
+
+```java
+// Create a BrowseRequest with the filter name and filter value to request results for
+BrowseRequest request = new BrowseRequest("group_id", "8193");
+
+// Add in additional parameters
+request.setResultsPerPage(5);
+request.setPage(1);
+request.setGroupId("625");r
+request.setSortBy("Price");
+request.setSortAscending(true);
+request.getFacets().put("Brand", Arrays.asList("Jif"))
+
+// Create a UserInfo object with the session and unique device identifier (optional)
+UserInfo userInfo = new UserInfo(5, "device-id-1123123");
+userInfo.setUserSegments(Arrays.asList("Desktop", "Chrome"));
+
+// Request results as an object
+BrowseResponse response = constructor.browse(request, userInfo);
+
+// Request results as a JSON string
+String response = constructor.browseAsJSON(request, userInfo);
+```
+
+If you'd like to retrieve browse results asynchronously, the above code can be modified slightly to utilize a callback methodology:
+
+```java
+constructor.browse(request, userInfo, new BrowseCallback() {
+  @Override
+  public void onFailure(final ConstructorException exception) {
+    // failure condition
+  }
+
+  @Override
+  public void onResponse(final SearchResponse response) {
+    // success condition - data located within `response`
+  };
+});
+```
+
+# Retrieving Recommendation Results
+
+To retrieve browse results, you will need to create a `BrowseRequest`. In this request, you can also specify the number of results you want and the items (given the ids) that you want to retrieve recommendations for. If the results are for a specific user, you can also create a `UserInfo` object, which will allow you to retrieve personalized results.
```

Comment:
Still sez `browse results` and `BrowseRequest`

Reply:
Updated, good catch!